src/calculateRentalCost.js

```diff
@@ -4,7 +4,21 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const dayPrice = 40;
+  const weekDiscount = 50;
+  const threeDayDiscount = 20;
+
+  let price = days * dayPrice;
+
+  if (days >= 7) {
+    price = price - weekDiscount;
+  }
+
+  if (days >= 3 && days < 7) {
+    price = price - threeDayDiscount;
+  }
```

Comment:
Since the first `if` condition includes a `return` statement, this second `if` does not need to check `days < 7` because it will only be reached if `days` is less than 7. Simplify the condition to just `days >= 3`.